//////////////////////////////////////////////////////////////////////////////
// WARNING: This code is machine generated. Manual changes will be overridden.

syntax = "proto3";

package octaneapi;

option optimize_for = CODE_SIZE;

import "common.proto";
import "google/protobuf/empty.proto";

 // GRPC proto definitions for 'ApiSceneOutliner' class from 'apisceneoutliner.h'
message ApiSceneOutliner
{
    // Request packet corresponding to '[in] parameters in ApiSceneOutliner::create'
    message createRequest
    {
    }

    // Response packet corresponding to '[out] parameters in 'ApiSceneOutliner::create'
    message createResponse
    {
        
        ObjectRef result = 1;
    }

    // Request packet corresponding to '[in] parameters in ApiSceneOutliner::destroy'
    message destroyRequest
    {
        // ID of the object on which to call the method
        ObjectRef objectPtr = 1;
    }

}


// GRPC interface definition for class 'ApiSceneOutliner' from 'apisceneoutliner.h'
service ApiSceneOutlinerService
{
    rpc create(ApiSceneOutliner.createRequest) returns (ApiSceneOutliner.createResponse);
    rpc destroy(ApiSceneOutliner.destroyRequest) returns (google.protobuf.Empty);
}
